wd_helper.au3

```diff
@@ -3221,6 +3221,12 @@ Func _WD_JsonActionKey($sType, $sKey, $iSuffix = Default)
 	Json_Put($vData, '.actions[0].type', $sType)
 	Json_Put($vData, '.actions[0].value', $sKey)
 	Local $sJSON = Json_Encode($vData)
+
+	; Don't encode backslash of Unicode character
+	If StringLeft($sKey,2) = '\u' Then
```

Comment:
hmm.... why not :
```
If StringLeft($sKey,2) = '\\u' Then
```
?

Reply:
`$sKey` is the incoming parameter from the calling routine, so the expectation is that this will be in the correct format (ie: \ue009). The proposed change is to fix the situation where the call to Json_Encode escapes the backslash by adding a second one. Therefore, I believe that the proposed change better addresses the situation.